src/spaceone/inventory/service/cloud_service_service.py

```diff
@@ -181,30 +176,42 @@ def update(self, params):
         old_cloud_svc_data = dict(cloud_svc_vo.to_dict())
 
         if 'tags' in params:
-            new_tags = self._create_tag(params, tag_type)
+            old_tags = old_cloud_svc_data.get('tags', {})
+            new_tags = self._create_tags(params['tags'], provider)
+            old_tag_keys = old_cloud_svc_data.get('tag_keys', {})
+            new_tag_keys = self._create_tag_keys(params['tags'], provider)
+
+            if new_tags != old_tags:
```

Comment:
Comparison of two tags should be limited to the input provider tags, not all tags.
`if new_tags[provider] != old_tags.get(provider, {}):`

Reply:
have reflected thank you.